configs/dev-config.yaml

```diff
@@ -21,6 +21,21 @@ asset:
 
 inspector_url: http://localhost:3333/dist/aframe-inspector.js
 
+default_data:
+  transitions:
+    [
+      {
+        "data":
+          [
+            {
+              "text": "INTRODUCTION: text goes here.",
+              "imageSrc": "image goes here",
+            },
+          ],
+      },
+      { "data": [{ "text": "CONCLUSION: just text, no images" }] },
+    ]
+
```

Comment:
Interesting that we have this in config, do we do this for any other fields? Because for now we can set mock data to have it, and then always assume (since we can control this) that all environments have these fields set.

Reply:
Talked with @jaydhulia about this, when we deploy the application.. when CDC has finalized the default text for intro and conclusion, we can update the config file and not have to modify the code.

But yeah for dev purposes.. it would not make a difference since we have our mockdata